napari/layers/shapes/shape_models/shape.py

```diff
@@ -252,9 +252,9 @@ def svg_props(self):
         """dict: color and width properties in the svg specification
         """
         width = str(self.edge_width)
-        face_color = (255 * self.face_color.rgba).astype(np.int)
+        face_color = (255 * self.face_color).astype(np.int)
```

Comment:
I think we might need to revert this too if we are reverting all the changes in the `shapes` folder, otherwise we might have some unexpected breakages (even though this assumes the vispy object representation)